sites.json

```diff
@@ -6009,7 +6099,17 @@
     },
 
     {
-        "name": "WAYN",
+        "name": "Walmart",
+        "url": "http://help.walmart.com/app/ask",
+        "difficulty": "impossible",
+        "notes": "Contact sutomer service to close your account.",
```

Comment:
Will Walmart close your account if you request it? If so, the difficulty should be "hard", not "impossible". Otherwise, this note isn't clear to me.